ui/package.json

```diff
@@ -30,12 +30,13 @@
     "react-use": "^17.3.2",
     "typescript": "^4.7.4",
     "use-nft": "^0.12.0",
-    "wagmi": "^0.4.2"
+    "wagmi": "^0.12.16"
```

Comment:
@yahiamarzouk Isn't wagmi 1.3.9 the newest version? - https://wagmi.sh/

Reply:
It is but then it doesn't work with ethersjs. Much of the code uses Ethersjs